integration/templates/cert_exporter_values.go

```diff
@@ -0,0 +1,12 @@
+// +build k8srequired
+
+package templates
+
+// CertExporterValues values used for cert-exporter-chart in e2e integration test.
+const CertExporterValues = `namespace: kube-system
+
+e2e: true
+
+image:
+  tag: [[ .SHA ]]
```

Comment:
Why do we need to do this? I know this is not set by architect but we shouldn't use that as it becomes obsolete and replaced by `[[ .Version ]]`. Still I don't get why we need to install it that way and can't install it from quay?

Reply:
Just for some context, I've:
- extracted formatted string literal of values https://github.com/giantswarm/cert-exporter/pull/39/files#diff-2aa5f36e3fcbb8680c0534589da3a183L111 into a separate package and constant
- done that as values constant seemed easier to read and maintain, and using the templates package+constant for values definition was consistent with other e2e integration tests that I've migrated so far
- had to add e2e key/value so PriorityClass resource gets installed just for the tests, assuming helm chart is not used for tests only but for actual cert-exporter deployment with PriorityClass managed/provisioned upfront elsewhere; with additional key/value test values overrides string literal would have become even less readable
- noticed that cert-exporter-chart's values.yaml uses same `[[ .SHA ]]` construct for image tag placeholder, and that existing test logic was setting the tag to the `env.CircleSHA()` value which can be seen again in https://github.com/giantswarm/cert-exporter/pull/39/files#diff-2aa5f36e3fcbb8680c0534589da3a183L111 - I assumed that that was there for a valid reason, wanted to make scope of changes focused to minikube to KIND migration only.

Now that you've raised concern, I gave it a try and optimized things a bit outside of pure migration, removed setting of `image.tag` in integration test's values overrides, and build+e2e workflow passes.

Please correct me if wrong, I guess that the correct cert-exporter image tag (one produced by the branch build) is actually being used in e2e test's helm release, `architect publish` step in e2eTest as one side-effect replaces `[[ .SHA ]]` reference in values.yaml of the helm chart so there's no need by the test logic to override the image.tag; same channel (sha+test_name) seems to be used by `architect publish` and test logic.